src/liballoc/tests/btree/map.rs

```diff
@@ -783,11 +796,11 @@ fn test_clone() {
         assert_eq!(map, map.clone());
     }
 
-    // Full 2-level and minimal 3-level tree (sizes 143, 144 -- the only ones we clone for).
-    for i in 1..=144 {
+    // Also test two big trees.
```

Comment:
The comment could still point out that cloning only actually happens for `MIN_INSERTS_HEIGHT_2-1` and `MIN_INSERTS_HEIGHT_2`.